kalite/updates/download_track.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from django.conf import settings
+logging = settings.LOG
+
+from .settings import VIDEO_DOWNLOAD_QUEUE_FILE
+
+class VideoQueue:
```

Comment:
@MCGallaspy the only change would be that the database would no longer be in the zip file. All the image resources would still be downloaded in the same way.